pkg/cloudproviders/aws/aws.go

```diff
@@ -101,16 +101,16 @@ func (p *Provider) Instance(ctx context.Context, IP ...gostatsd.IP) (map[gostats
 				if err != nil {
 					log.Errorf("Error getting instance region: %v", err)
 				}
-				tags := make(gostatsd.Tags, len(instance.Tags))
+				tags := make(gostatsd.Tags, len(instance.Tags)+1)
 				for idx, tag := range instance.Tags {
 					tags[idx] = fmt.Sprintf("%s:%s",
 						gostatsd.NormalizeTagKey(aws.StringValue(tag.Key)),
 						aws.StringValue(tag.Value))
 				}
+				tags[len(instance.Tags)-1] = fmt.Sprintf("region:%s", region)
```

Comment:
String concatenation should be cheaper than formatting?

Reply:
Yep, this isn't the expensive path though.  I'll make a note to fix it later.